packages/web/src/fixtures/dev-kit/hooks.ts

```diff
@@ -24,12 +14,23 @@ export const useGetTotalRewardsAmount = (propertyAddress: string) => {
   return { totalRewardsAmount: data ? toNaturalNumber(data) : undefined, error }
 }
 
-const getTotalStakingAmount = async (proepertyAddress: string) => {
-  const client = newClient()
-  if (client) {
-    return client.lockup(await client.registry(addresses.eth.main.registry).lockup()).getPropertyValue(proepertyAddress)
+export const useWithdrawHolderReward = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [error, setError] = useState<Error | undefined>(undefined)
```

Comment:
I don't think of "Error" instance as a "state", so I don't want to manage it in a state. 
I'd like to state an Error message instead of it, what do you think? And do we need to manage the Error as a state?

Reply:
I'm sorry. Since this is a hooks wrapped API, I thought it would be better to return the error instance as is. It will be fine as it is.